### ISSUE TITLE
How to deal with attention_mask?

### ISSUE THREAD

**Issue by lukeewin** [QwenLM/Qwen1.5#267]
当我运行时提示：The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation.

请问这个如何解决？

**Comment by jklj077** [comment]
In general, you can simply ignore this warning. 

Or do something like this in inference code
```python
inputs = tokenizer(my_awesome_text, return_tensors="pt") # this contains attention_mask
outputs = model.generate(**inputs, **my_awesome_generation_parameters)
```

If you have encountered other problems, please be more specific.
